pkg/test/framework/components/istio/istio.go

```diff
@@ -46,6 +46,12 @@ func Setup(i *Instance, cfn SetupConfigFn, ctxFns ...SetupContextFn) resource.Se
 			if err != nil {
 				return err
 			}
+			if ctx.Environment().IsMulticluster() {
+				// Only setup namespaces in multicluster, some tests still rely on the default namespace config
+				if err := setupNamespaces(ctx, &cfg); err != nil {
```

Comment:
I'd like to change this to something that explicitly says to use generated namespaces.